Makefile

```diff
@@ -196,13 +199,12 @@ $(RESOURCES_DOT_GO): $(NPM)
 	git update-index --assume-unchanged $$APP/js/revision.js && \
 	DEST="$@" && \
 	cd $$LANTERN_UI && \
-	npm install && \
+	$(NPM) install && \
 	rm -Rf dist && \
-	gulp build && \
+	$(GULP) build && \
 	cd - && \
 	rm -f bin/tarfs && \
-	go install github.com/getlantern/tarfs/tarfs && \
-	mv bin/tarfs.exe bin/tarfs ; \
+	go build -o bin/tarfs github.com/getlantern/tarfs/tarfs && \
```

Comment:
When using go1.6 for x64, the `go build github.com/getlantern/tarfs/tarfs` generated a binary named `bin/windows_386/tarfs.exe` (because the `GOARCH` was set to 386), instead of fiddling around it we can force the name to be `bin/tarfs`.